src/vtk_calculator.py

```diff
@@ -65,31 +70,51 @@ def parse_args():
     return args
 
 
+twoDFunctions = {
+    "franke2d": """0.75*exp(-((9*{first}-2)^2+(9*{second}-2)^2)/4)
+    +0.75*exp(-(9*{first}+1)^2/49-(9*{second}+1)/10)
+    +0.5*exp(-((9*{first}-7)^2+(9*{second}-3)^2)/4)
+    -0.2*exp(-(9*{first}-4)^2-(9*{second}-7)^2)""",
+    "eggholder2d": """-{first}*sin(sqrt(abs({first}-{second}-47)))
+    -({second}+47)*sin(sqrt(abs(0.5*{first}+{second}+47)))""",
```

Comment:
Using the `"""` includes newlines in the string, which messes up the output.

Python automatically concatenates following strings. So you can write this:
```suggestion
    "franke2d": "0.75*exp(-((9*{first}-2)^2+(9*{second}-2)^2)/4)"
    "+0.75*exp(-(9*{first}+1)^2/49-(9*{second}+1)/10)"
    "+0.5*exp(-((9*{first}-7)^2+(9*{second}-3)^2)/4)"
    "-0.2*exp(-(9*{first}-4)^2-(9*{second}-7)^2)",
    "eggholder2d": "-{first}*sin(sqrt(abs({first}-{second}-47)))"
    "-({second}+47)*sin(sqrt(abs(0.5*{first}+{second}+47)))",
```

Reply:
I was not aware of automatic concatenation, thank you